### DISCUSSION TITLE
[v5] - node ace build don't build css file

### DISCUSSION THREAD

**Discussion by ekimkael** [adonisjs/core#4553] [Help] [answered]
Hi guys,
I'm using adonisjs v5 on a project and when I try to build or to set entryPointStyle('app') my css file is not taking into account.
The build command doesn't add css in entrypoint.json and the <link> is not added to the head.

I didn't change anything related to assets compilation or webpack.config.js.
I just removed the default styles on app.css and added my own

**Comment by angelocoutinho** [comment, marked as answer]
Hey there,

I had the same issue with AdonisJS v5 where my CSS wasn't being included after building or using `@entryPointStyle('app')`. Here's how I fixed it:

### Configure Webpack Encore
First, run the following command to set up Encore:
```bash
node ace configure encore
```

### Create Entry Point Files
Make sure you have `resources/js/app.js` and `resources/css/app.css` files. If they don't exist, create them.

### Import CSS in JavaScript
In `resources/js/app.js`, add this line:

```javascript
import '../css/app.css';
```

### Update Your Views
Include entry point scripts and styles in your HTML template. Your view should look like this:

```edge
<!DOCTYPE html>
<html lang="en">
<head>
  @entryPointScripts('app')
  @entryPointStyles('app')
</head>
<body>
</body>
</html>
```
### Run the Build Command
Run your app with the --watch flag:

```bash
node ace serve --watch
```

These steps should get your CSS working. More details are in the [AdonisJS v5 docs](https://v5-docs.adonisjs.com/guides/assets-manager).

Hope this helps!

**Reply by ekimkael** [reply]
Thank you so much, it works!!!
